src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -541,6 +542,7 @@ private void InitializePrecompiledContracts()
             [MapToG2Precompile.Address] = new(MapToG2Precompile.Instance),
 
             [PointEvaluationPrecompile.Address] = new(PointEvaluationPrecompile.Instance),
+            //[BeaconBlockRootPrecompile.Address] = new(BeaconBlockRootPrecompile.Instance),
```

Comment:
No need in precompile I guess

Reply:
@Demuirgos could you confirm?

Reply:
oh you, it can be removed, I think I already removed most usages of the ``BeaconBlockRootPrecompile`` class